cobbler/api.py

```diff
@@ -637,9 +633,9 @@ def set_item_resolved_value(
             return
         # Deduplicate - only for dict
         if isinstance(property_object_of_attribute, InheritableDictProperty):
-            parent_item = desired_item.logical_parent
-            if hasattr(parent_item, attribute):
-                parent_value = getattr(parent_item, attribute)
+            parent_item = desired_item.logical_parent  # type: ignore
```

Comment:
I think we should leave the comment for now as the logic will be reworked with the direct next follow-up PR that we already have in the queue. As such I would like to not perform what you are proposing.

If you agree with that please resolve the conversation.

Reply:
I'm fine with leaving the ignores here, since we'll change the `Item` to `InheritableItem` in the future, but the casting will still be required AFAIK. We might want to add a `TODO` comment here as well? 

If you know you won't forget about this, I'm OK with resolving and keeping as is :) Personally, I'm quite sure I'll forget though :)

Reply:
Adding a TODO comment with the next push